Valigator.AspNetCore.Newtonsoft.Json/MvcBuilderExtensions.cs

```diff
@@ -58,5 +62,10 @@ public static IMvcBuilder AddValigator(this IMvcBuilder builder, Func<AspNetCore
 				.AddJsonOptions(o => o.SerializerSettings.Converters.Add(new ValigatorConverter()));
 #endif
 		}
+
+		private class NullObjectModelValidator : IObjectModelValidator
```

Comment:
This could be put into `Valigator.AspNetCore`, but due to the fact that it may change independent of the System.Text version of this, I thought putting a `NullObjectmOdelValidator` in both places was prudent.